### PR TITLE
Implement/express server

### PR DESCRIPTION
**Pull request by Busyreadingsomething**
Ticket: https://trello.com/c/9NHb7h8t/11-create-basic-server-with-express-m

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
 module.exports = {
-    "extends": "airbnb"
-};
\ No newline at end of file
+  "extends": "airbnb",
+  // "rules": {
```
Comment (beth) [inline .eslintrc.js]: Please remove.

File: package.json [hunk 2]
```diff
@@ -41,6 +41,8 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.7.0",
     "jest": "^22.4.0",
+    "nodemon": "^1.15.1",
```
Comment (beth) [inline package.json:45]: Either use this in the PR or remove.

File: spec/server.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const app = require('../server/app');
+
+describe('Test the root path', () => {
+  test('It should response the GET method', () =>
```
Comment (beth) [inline spec/server.test.js]: "response" => "respond to"

File: spec/server.test.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const app = require('../server/app');
+
+describe('Test the root path', () => {
```
Comment (beth) [inline spec/server.test.js:4]: This doesn't seem like an accurate title - only two of the tests are testing the root path. I would include the word server or api in this test as well.

File: spec/server.test.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  test('It should response the GET method', () =>
+    request(app).get('/').then(response =>
+      expect(response.statusCode).toBe(200)));
+
+  test('It should get a response Hello World back', () =>
```
Comment (beth) [inline spec/server.test.js]: Tests on line 9 and 13 will be obsolete in next feature, most likely. Consider either not including OR including data + paths that match what you actually want to build.

**Review by beth — CHANGES_REQUESTED** [review]
Mac,

Overall looks good, but I would avoid writing throwaway tests that you already know you're going to change. 

Additionally, make sure to remove extraneous comments and dependences that aren't actually needed for the feature in the PR.

**Comment by Busyreadingsomething** [comment]
@nicola24 fixed the changes. Please review when you get the chance.

**Review by nicola24 — APPROVED** [review]
Good to go, make sure to get rid of the console log

**Pull request merged**